src/IsolatedRidgeCorrectionFactor.cpp

```diff
@@ -16,25 +16,47 @@ double IsolatedRidgeCorrectionFactor(double d1_hzn__km, double d2_hzn__km, doubl
     double d_1__km[3] = { 15.0, 30.0, 60.0 };
     double d_2__km[9] = { 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 };
 
-    // points from Figure 31, Okumura
+    // points from Figure 31, Okumura, at corresponding d2 distances
     double curve_data[3][9] =
             { {  4.0, -13.0, -17.5, -17.5, -15.0, -12.5, -10.0, -8.0, -6.0 },   // C curve : d1 <= 15 km
               { 12.0,  -8.5, -13.0, -12.0, -10.0,  -8.0,  -6.5, -5.0, -4.0 },   // B curve : d1 <= 30 km
               { 20.0,  -4.0,  -6.5,  -6.0,  -4.5,  -3.5,  -2.5, -2.0, -1.0 } }; // A curve : d1 <= 60 km
 
     // normalized ridge height factor
-    double alpha = sqrt(h_edge__meter / 200.0); // Eq 1, Okumura, alpha = 0.07 * sqrt(h)
+    // Eq 1, Okumura, alpha = 0.07 * sqrt(h)
+    double alpha = sqrt(h_edge__meter / 200.0);
```

Comment:
May be helpful to add comment to indicate height is being normalized to 200 meters which approx equals square root of 1/200